content/docs/extension-guide/create-extension/advanced-extensions.md

```diff
@@ -0,0 +1,155 @@
++++
+title="The finer points of image extensions"
+weight=406
++++
+
+# Guidance for extension authors
```

Comment:
@AidanDelaney I'd love to hear your thoughts about how this page should be organized.

Reply:
Let's leave it here for now, as the "extension-guide" section isn't for a particular persona, unlike the "buildpack author guide" or the "platform guide"